packages/governance/src/contractGovernor.js

```diff
@@ -0,0 +1,87 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+import { E } from '@agoric/eventual-send';
+import { details as X } from '@agoric/assert';
+
+import { setupGovernance } from './governParam';
+
+/*
+ * ContractManager is an ElectionManager that starts up a contract and hands its
+ * own creator a facet that allows them to call for votes on parameters that
+ * were declared by the contract.
+ *
+ * ContractManager is initialized by a call to startGovernedInstance() on this
+ * contract's creatorFacet. That call initializes the registrar, and identifies
```

Comment:
This sounded like it was creating the registrar, but then I realized it meant that the registrar is saved in the contract.